### ISSUE TITLE
Broken hint button

### ISSUE THREAD

**Issue by mitm001** [jMonkeyEngine/sdk#105]
When using the SDK the hint button will mysteriously break when needing to add an import statement. 

The mouseover shows the problem but the hint button no longer works. It seems to occur after deleting things then using Source/Fix Imports to clean things up. Sometimes using the Source/Fix Imports button will add the statements when this happens, other times it wont. Even if Source/Fix Imports does work, the button will continue to be disabled when needing a new import and nothing seems to fix it. Sometimes it clears up on its own. However, when it doesn't then that particular app will continue to experience the problem.

Its being a real pain to try and duplicate.

**Comment by MeFisto94** [comment]
This sounds like an actual problem with Netbeans itself? I did not experience these issues however I am always using CTRL+SHIFT+I for the imports (which is also my recommended workaround).

If you manage to trigger that again a Screenshot would be nice to maybe give us a hint.
Also try to change the "Theme" back to "Netbeans" (instead of Darkmonkey), to see if the Theme is messing things up

**Comment by mitm001** [comment]
I already use "NetBeans" theme.

I had other issues that have occurred since this post. All the JDK settings I had made went poof except for the look and feel and the profile. Although I could reapply them, when exiting the IDE they would not stay saved. I have been using the SDK for about a month with no problems except the one mentioned above. I had to re-install the SDK to get things to work normal again. It appears to me somehow the program became corrupted. I wont be using fix imports again....
